widgets/actionsMenu.widget.js

```diff
@@ -42,6 +43,17 @@ class myWidget extends baseWidget() {
     })
   }
 
+  deleteSelectedContainer () {
+    if (this.widgetsRepo && this.widgetsRepo.has('containerList')) {
+      const containerId = this.widgetsRepo.get('containerList').getSelectedContainer()
+      if (containerId && containerId !== 0 && containerId !== false) {
+        this.utilsRepo.get('docker').removeContainer(containerId, () => {
+          // TODO: emit an event for a refreshed list of containers and images
```

Comment:
Could you also add here the message event for the status box that the container was removed?

Reply:
The message couldn't display and I don't know why exactly. Actually I hoped that you could do it 🙄
I don't have much time to find the reason also, sorry

Reply:
No worries then, I'll take it up.